### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by craigklem**
Had to change exclude to get it to work for me.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ config.set({
             loaders: [ ... ],
             postLoaders: [ { // << add subject as webpack's postloader
                 test: /\.js$/,
-                exclude: /(test|node_modules|bower_components)\//,
+                exclude: [/test/, /node_modules/, /bower_components/],
```
Comment (nkbt) [inline README.md]: I use `[path.resolve('test')]` which is crossplatform and looks nice

Comment (deepsweet) [inline README.md]: same here, I just need to update README like isparta-loader https://github.com/deepsweet/isparta-loader#karmaconfjs

**Comment by deepsweet** [comment]
fixed, thanks for reminder!

**Comment by craigklem** [comment]
Nice fix. Thanks!

**Pull request closed without merging**